### ISSUE TITLE
Wolf Hound gets asked to choose their allegiance every round even after choosing

### ISSUE THREAD

**Issue by ajesler** [hjwylde/werewolf#96]
Wolf Hound gets asked to choose their allegiance every round, even if they have already chosen one.

```
Waiting on you...
Which allegiance do you 'choose' to be aligned with?
Whom would you like to 'vote' to devour?
```

**Comment by hjwylde** [comment]
Since v0.4.7.0.

**Comment by hjwylde** [comment]
Fixed in v0.4.7.1.
